runtime/autoload/provider/clipboard.vim

```diff
@@ -101,9 +101,9 @@ function! provider#clipboard#Executable() abort
     let s:copy['*'] = s:copy['+']
     let s:paste['*'] = s:paste['+']
     return 'doitclient'
-  elseif executable('win32yank')
-    let s:copy['+'] = 'win32yank -i --crlf'
-    let s:paste['+'] = 'win32yank -o --lf'
+  elseif executable('win32yank.exe')
```

Comment:
This doesn't need to change because `executable` iterates through $PATHEXT. 

And `executable('win32yank.exe')` is a bit misleading because it will succeed if it finds `win32yank.exe.exe`, `win32yank.exe.bat`, etc. So it's slightly better to use `executable('win32yank')`.

Reply:
On my Linux system, with `win32yank.exe` in `$PATH`, `executable('win32yank.exe')` returns 1, while `executable('win32yank')` returns 0. While a bit of a kludge, using `win32yank.exe` in the check makes it so that it passes on both Windows and WSL.

Alternatively, we could add checks like `(linux && win32yank.exe) || (windows && win32yank)`, if you think it is cleaner.

Reply:
Gah, I was thinking about Windows. PATHEXT is not iterated on WSL, of course.